src/plugins/auto/plugin.cpp

```diff
@@ -61,6 +62,7 @@ namespace {
                     res.push_back(PluginConfigParams::KEY_PERF_COUNT);
                     res.push_back(PluginConfigParams::KEY_EXCLUSIVE_ASYNC_REQUESTS);
                     res.push_back(MultiDeviceConfigParams::KEY_AUTO_NETWORK_PRIORITY);
```

Comment:
my understanding that the model_priority should replace (or alias?) the ov::hint:model_priority.
cc  @apankratovantonp

Reply:
Yes, will update soon.